crates/starfish/core/src/transactions_synchronizer.rs

```diff
@@ -279,19 +280,19 @@ impl InflightTransactionsMap {
     /// has been called, it will panic.
     fn unlock_transactions(
         self: &Arc<Self>,
-        block_refs: &BTreeSet<BlockRef>,
+        block_refs: &BTreeSet<GenericTransactionRef>,
```

Comment:
block_refs doesn't fit anymore. Also, gen_tr_ref is a bit weird, I'd call it tx_refs and tx_ref correspondingly.